tests/pathfinding/conftest.py

```diff
@@ -44,12 +44,17 @@ def no_greenlets_left():
     yield
     tasks = _get_running_greenlets()
     # give all tasks the chance to clean themselves up
+    for task in tasks:
```

Comment:
Puh, this stuff is ugly

Reply:
yes it is, let's change it next week with Ulo :)